onadata/apps/api/management/commands/retrieve_org_or_project_list.py

```diff
@@ -0,0 +1,133 @@
+import json
+from django.core.management.base import (
+    BaseCommand, CommandError, CommandParser)
+from django.utils.translation import gettext as _
+
+from onadata.apps.logger.models import Project
+from onadata.apps.api.models import OrganizationProfile
+from onadata.libs.permissions import is_organization, get_role
+from onadata.libs.serializers.organization_serializer import \
+    OrganizationSerializer
+
+
+def get_project_users(project):
+    ret = {}
+
+    for perm in project.projectuserobjectpermission_set.all():
+        if perm.user.username not in ret:
+            user = perm.user
+
+            ret[user.username] = {
+                'permissions': [],
+                'is_org': is_organization(user.profile),
+                'first_name': user.first_name,
+                'last_name': user.last_name,
+            }
+
+        ret[perm.user.username]['permissions'].append(perm.permission.codename)
+
+    for user in ret.keys():
+        ret[user]['permissions'].sort()
```

Comment:
Is there a specific reason we are sorting this?

Reply:
Not really... I've changed this in the latest commit